app/views/planning_application/local_policies/_comment.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if @local_policy.review_local_policies.where.not(reviewer_comment: "").any? %>
```

Comment:
This shouldn't be in a view - it should be an extension on the review_local_policies association or a method on local_policy. Also is the query `reviewer_comment = '' OR reviewer_comment IS NULL` ? If so and the association is already loaded it might be quicker to do in memory by `review_local_policies.any?(&:present?)`.